.github/workflows/e2e.yaml

```
@@ -150,15 +150,17 @@ jobs:
           path: ${{ inputs.frontendRepo }}
           token: ${{ secrets.token }}
 
-      - name: Use Node.js 12.18.0
+      - name: Use Node.js
         if: steps.cache.outputs.cache-hit != 'true'
         uses: actions/setup-node@v2
         with:
-          node-version: 12.18.0
+          cache: 'yarn'
+          cache-dependency-path: './${{ inputs.frontendRepo }}/yarn.lock'
+          node-version-file: './${{ inputs.frontendRepo }}/.nvmrc'
 
       - name: Install dependencies
         if: steps.cache.outputs.cache-hit != 'true'
-        run: npm install yarn && yarn install && yarn add serve
+        run: yarn install && yarn add serve
```

nb. this should really be a global add instead of adding it to the project dependencies. Doing the latter is not ideal because it can change the resolution for actual dependencies in the project as Yarn tries to deduplicate overlapping ranges in yarn.lock

```suggestion
        run: yarn install && yarn global add serve
```

https://github.com/vercel/serve#usage

---

This didn't work. `error Command "serve" not found.`